packages/state-manager/src/utils/tests/types.ts

```diff
@@ -1,7 +1,17 @@
-import { configureStore, combineReducers } from '@reduxjs/toolkit'
+import { configureStore, combineReducers, getDefaultMiddleware } from '@reduxjs/toolkit'
 import { reducers } from './prepareStore'
+import createSagaMiddleware from 'redux-saga'
+
 const rootReducer = combineReducers(reducers)
-const store = configureStore({ reducer: rootReducer })
+const sagaMiddleware = createSagaMiddleware()
+
+const store = configureStore({
+  reducer: rootReducer,
+  middleware: [
```

Comment:
unless middlewares are the same, the type of dispatch mismatch. By default thunkMiddleware is enabled so it was giving me grief on few places there type `Store` was used against the one declared in desktop package